### ISSUE TITLE
Spoiler tags

### ISSUE THREAD

**Issue by badspott** [bluesky-social/social-app#5840]
### Describe the Feature

The app already has quite a lot of useful warnings (nsfw, etc) but one for 'spoilers' would be nice. Especially for people who like to post things from series, movies, books, or games so they can properly spoiler tag their images or even text.

**Comment by NotCassim** [comment]
I second that. A spoiler label would be very useful to blur stuff out.

**Comment by Azuretare** [comment]
I really wanna share music from a game series that I love but don't wanna spoil 1 of my followers that I know is not there yet! (The name of the song is a spoiler but sometimes the thumbnail can be too)

**Comment by M-Osky** [comment]
This will improve quite a lot the way we use the App, frequently I don't post about series or comics I'm reading to not spoil them for the people that is not up to date.
Likewise when a I'm not up to date with a series that is airing I frequently avoid the Discover tab to avoid Spoilers.
I feel like the tools are already there and should be quite straight-forward

**Comment by 4Bakers** [comment]
I'm posting a picture of a very corroded AA battery, for example.
it's not explicit, it's not NSFW, but it's unpleasant to look at.
I'd love to apply a spoiler tag to that!

**Comment by jphastings** [comment]
I'd like to be able to tag spoilers with an ID, so that I can have spoilers for things I've already watched/read/visited/done be auto-shown.

**Comment by Reonu** [comment]
Seconded, I missed this back in the Twitter days and I miss it now on Bluesky

**Comment by nektro** [comment]
I'd much prefer something like `|| spoiler ||` as a few other platforms support, rather than a binary switch on the entire post being a spoiler

**Comment by NotCassim** [comment]
up

**Comment by thirashima** [comment]
Yes please. Something like the content warning system from Mastodon would be great.

**Comment by mmoustafa8108** [comment]
up up up :)

**Comment by IArnaut2** [comment]
@bluesky-social Add spoiler tag, you stubborn liberal techbros.
